src/main/java/de/dataelementhub/rest/controller/v1/ExportController.java

```diff
@@ -0,0 +1,96 @@
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/v1")
+public class ExportController {
+
+  private final ExportService exportService;
+
+  @Autowired
+  public ExportController(ExportService exportService) {
+    this.exportService = exportService;
+  }
+
+  public static String exportDirectory = System.getProperty("user.dir") + "/uploads/export/";
+
+  /**
+   * Return an overview of all exports.
+   */
+  @Order(SecurityProperties.BASIC_AUTH_ORDER)
+  @GetMapping(value = "/export")
+  public ResponseEntity<List<ExportInfo>> allExports() {
+    Integer userId = DataElementHubRestApplication.getCurrentUser().getId();
+    List<ExportInfo> exportDescriptions = exportService.allExports(userId, exportDirectory);
+    return new ResponseEntity<List<ExportInfo>>(exportDescriptions, HttpStatus.ACCEPTED);
+  }
+
+  /**
+   * Start an export process for all given elements.
+   */
+  @Order(SecurityProperties.BASIC_AUTH_ORDER)
+  @PostMapping(value = "/export")
+  public ResponseEntity<String> export(@RequestBody ExportRequest exportRequest,
+      @RequestParam String format, @RequestParam Boolean fullExport) {
+    int numberOfNamespacesExportedFrom = exportRequest.getElementUrns().stream()
+        .map(e -> e.split(":")[1]).collect(Collectors.toSet()).size();
+    if (numberOfNamespacesExportedFrom > 1) {
+      return new ResponseEntity<>("Export from more than one namespace is forbidden",
+          HttpStatus.NOT_ACCEPTABLE);
+    }
+    Integer userId = DataElementHubRestApplication.getCurrentUser().getId();
+    String timestamp = new Timestamp(System.currentTimeMillis())
+        .toString().replaceAll("[ \\.\\-\\:]", "_");
+    exportService
+        .exportService(exportRequest, userId, format, fullExport, timestamp, exportDirectory);
+    return new ResponseEntity<>(timestamp, HttpStatus.ACCEPTED);
+  }
+
+
+  /**
+   * Returns export file if done otherwise returns export status.
+   */
+  @GetMapping(produces = "application/zip", value = "/export/{exportId}")
+  @Order(SecurityProperties.BASIC_AUTH_ORDER)
+  public ResponseEntity exportStatus(
+      @PathVariable(value = "exportId") String exportId)
+      throws Exception {
+    Integer userId = DataElementHubRestApplication.getCurrentUser().getId();
+    ExportInfo exportInfo = exportService.exportInfo(exportId, userId, "export");
+    switch (exportInfo.getStatus()) {
+      case "NOT DEFINED":
+        return new ResponseEntity<>(exportInfo.toString(), HttpStatus.NOT_ACCEPTABLE);
```

Comment:
What exactly does "not defined" mean here? From [dataelementhub.model PR 12](https://github.com/mig-frankfurt/dataelementhub.model/pull/12/files), file ExportService (src/main/java/de/dataelementhub/model/service/ExportService.java)) I guess it means that no export with this id exists.

If this is the case - once more, "NOT ACCEPTABLE" is not the correct status code to send - a simple 404 NOT FOUND should be correct here.